### ISSUE TITLE
Show the attributes of a node on click?

### ISSUE THREAD

**Issue by networkgraph** [jacomyal/sigma.js#599]
Hi, I was surprised that I was unable to find a plug-in or capability to simply click a node and have information about the node be shown.  I also did not find any instances in which the question was previously asked in this forum.  I am very new to programming so perhaps there is an extraordinarily obvious way through which this can be done with an existing plugin, but nevertheless I am unable to find it.

So - is there a simple way to do this with the existing sigma.js code?  If not, could you advise me on how to go about making it?  I noticed that  the neighborhoods and the drag-nodes plugins have the ability to listen to click nodes events, so maybe the best approach would be to adapt one of those?

**Comment by Yomguithereal** [comment]
Hello @networkgraph. Did you solve your problem? A first approach would indeed be to listen to the clickNode event and react accordingly.

**Comment by networkgraph** [comment]
Thanks for looking into it.  I am closer but still not there.  I can't figure out where the reaction to the clickNode event is defined.  I'm using an adaption of the filter.html example for the filter plugin.  At first glance, it seems as though the reaction clickNode event is in src/misc/sigma.misc.bindEvents.js.  At the bottom of that file, there are a few lines that seem to control which functions are used for a given mouse event.  But I tried to comment out these lines to see if this is indeed how I would change the reaction to a clickNode event and it didn't actually have any effect when you click a node - whether or not you comment out the "captor.bind('click', onClick);", the node label appears when you click a node.  What's funny is that commenting out the "captor.bind('mousemove', onMove);" line does change whether or not a label appears when you hover over a node.  I included the sectio nI'm talking about below.

So anyway, I'm not actually at the point where I am writing my function to react to the clickNode event; I'm doing all of this just to figure out where I should be putting that function.  Again, total rookie here.
'''javascript
// Bind events:
      //captor.bind('click', onClick);
      captor.bind('mousedown', onMove);
      //captor.bind('mouseup', onMove);
      captor.bind('mousemove', onMove);
      //captor.bind('mouseout', onOut);
      //captor.bind('doubleclick', onDoubleClick);
      //captor.bind('rightclick', onRightClick);
      //self.bind('render', onMove);
    }
'''
